public/app/features/alerting/unified/components/rules/state-history/useRuleHistoryRecords.tsx

```diff
@@ -119,18 +119,27 @@ export function logRecordsToDataFrame(
           color: { mode: 'thresholds' },
           custom: { fillOpacity: 100 },
           mappings: [
+            {
+              type: MappingType.RegexToText,
+              options: {
+                //  Map as a regex so we capture `Normal`, and `Normal (Updated)`
+                pattern: '/Normal/',
```

Comment:
nit: maybe `/^normal/i`? The current regex would match something like `Firing (not Normal)` (which doesn't exist, but perhaps we want to err on the safer side?)